folder/subfolder/out.csv

```diff
@@ -0,0 +1,3 @@
+,name,mask,weapon
```

Comment:
why have we created a csv file?

Reply:
ah, this was by mistake.. will remove this